server/service/src/invoice_line/stock_out_line/insert/mod.rs

```diff
@@ -73,7 +78,17 @@ pub fn insert_stock_out_line(
         .connection
         .transaction_sync(|connection| {
             let (item, invoice, batch) = validate(&input, &ctx.store_id, connection)?;
```

Comment:
getting full invoice object (from invoice repo vs invoice_row repo) would give the name_row, can be used to get name_id and name row type = patient